apps/AppWithWearable/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -711,6 +711,7 @@
 				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CODE_SIGN_ENTITLEMENTS = OneSignalNotificationServiceExtension/OneSignalNotificationServiceExtension.entitlements;
+				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
Can you revert those changes? The are not needed for your PR.

Reply:
I'm not really sure what changes caused those values to be added to the pbxproj file. So, can't really revert those changes. I guess Joan running the app locally will override these changes